waves-ext/src/main/resources/application.conf

```diff
@@ -11,6 +11,8 @@ waves.dex {
   }
 }
 
+waves.extension.dex.allow-tx-rebroadcasting = true
```

Comment:
Even we enable this functionality, it won't work without https://github.com/wavesplatform/Waves/blob/version-1.2.x/node/src/main/resources/application.conf#L263
So I suggest to remove `waves.extension.dex.allow-tx-rebroadcasting`.
Because the NODE's `allow-tx-rebroadcasting` is true by default, we won't need to add this to our configuration.

Reply:
Fixed